schemas/com.amazon.aws.lambda/java-context/jsonschema/1-0-0

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema" : "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "self" : {
+    "vendor" : "com.amazon.aws.lambda",
+    "name" : "java-context",
```

Comment:
I think we should use `_` for consistency with the other `com.amazon.aws.*` schema(s)